### PR TITLE
Delete react button v2

### PR DESCRIPTION
**Pull request by abacus2020**
Functioning Delete Button as a React component

Behaviour:
- Click delete
- See message 'Delete Successful'
- Refresh Page
- Post removed from feed

### CODE REVIEW

File: controllers/posts.js [hunk 1]
```diff
@@ -33,16 +33,26 @@ const PostsController = {
       res.status(201).redirect("/posts");
     });
   },
-  Delete: (req, res) => {
+  // Delete: (req, res) => {
+  //   Post.findOneAndDelete({ _id: req.params.id }).exec(function (err) {
```
Comment (sophiegilder) [inline controllers/posts.js:37]: Can this be deleted?

File: src/delete_button.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  render() {
+    const { status } = this.state;
+    return <button onClick={this.addDelete}>{status}</button>;
+  }
+}
+
+// state data set
+
+// let domContainer = document.querySelector(".delete-button-container");
```
Comment (sophiegilder) [inline src/delete_button.js:24]: Does this also need deleting?

**Review by robimaliqi — APPROVED** [review]
approved

**Review by sophiegilder — CHANGES_REQUESTED** [review]

**Review by edra-dev — APPROVED** [review]

**Review by BenjaminNeustadt — APPROVED** [review]
commentes deleted

**Review by BenjaminNeustadt — APPROVED** [review]
removed comments

**Review by sophiegilder — APPROVED** [review]

**Pull request merged**